packages/lex-cli/src/codegen/lex-gen.ts

```diff
@@ -306,12 +303,9 @@ export function genXrpcInput(
   lexUri: string,
   defaultsArePresent = true,
 ) {
-  const def = lexicons.getDefOrThrow(lexUri, [
-    'query',
-    'procedure',
-  ]) as LexXrpcProcedure
+  const def = lexicons.getDefOrThrow(lexUri, ['query', 'procedure'])
```

Comment:
There might be a bug in the code here as the type casting did not match the expected types. Not sure what is expected here.